expyfun/_experiment_controller.py

```diff
@@ -12,11 +12,6 @@
 from os import path as op
 from functools import partial
 import traceback as tb
-try:
-    import pyglet
-    from pyglet import gl
-except Exception:  # maybe we don't have a display (e.g., in terminal)
-    pyglet = gl = None
```

Comment:
It should make `import expyfun` work even on systems without a display (i.e., the OSX Travis CI). And it has the advantage of making all `expyfun` sub-imports, like to `expyfun.analyze` or `stimuli` faster. It only slows down and creates an OpenGL context when necessary, i.e. when instantiating `ExperimentController`